kafka_consumer/check.py

```diff
@@ -166,6 +168,13 @@ def check(self, instance):
         consumer_offsets = self._get_zk_consumer_offsets(
             zk_hosts_ports, consumer_groups, zk_prefix)
 
+        if len(consumer_offsets) > CONTEXT_UPPER_BOUND:
```

Comment:
I think it'd make sense to have a (hidden) config parameter to customize this upper bound

Reply:
\+ let's add a warning about the upper bound in the example file so that users are aware of this limitation before they start using the check :)